WhoopTriggerzDesign/CustomViews/CustomSliderModifier.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  CustomSliderModifier.swift
+//  WhoopTriggerzDesign
+//
+//  Created by Jessica Perez on 5/10/23.
+//
+import SwiftUI
+
+struct CustomSliderModifier: ViewModifier {
+    //MARK: - Properties
+    @Binding var value : CGFloat
+    @State var isMuted: Bool = false
+    @State var speakerImage: String
+//MARK: -Body
+    func body(content: Content) -> some View {
+        HStack {
+            // Speaker Button
+            Button(action: {
+                // Speaker toggle button
+                isMuted.toggle()
+                speakerImage = isMuted ? "Mute" : "Volume"
+                //Check mute button 
+                print(isMuted ? "Muted" : "Unmuted")
+                 
+            }) {
+                Image(speakerImage)
```

Comment:
Instead of having the `speakerImage` state variable we had above, because `isMuted` is already a state variable, it will also cause the view to be redrawn whenever it's changed. On line `20`, we are already toggling the `isMuted` variable. Because of that, we can do the line I suggested below and it will work the same.
```suggestion
                Image(isMuted ? "Mute" : "Volume")
```

Reply:
Resolved in 1d0cfdf